src/__e2e__/publishE2E.test.ts

```diff
@@ -54,6 +54,7 @@ describe('publish command (e2e)', () => {
 
```

Comment:
Any test coverage worth adding for the _password flow?

Reply:
I had thought about it I don't see any pre-existing token e2e tests. The logic I added is actually pretty straightforward, but if Ken feels strongly about it we probably need to figure out how to add baseline tests as well.

Reply:
Nah, this is fine. If you want to add a test - it would only be the getNpmAuthArgs function, but that's like 90% just a code move. So I don't mind it not having tests (yet?)